docs/src/main/asciidoc/secret-management.adoc

```diff
@@ -14,165 +19,188 @@ spring-cloud-azure-starter-keyvault-secrets adds Azure Key Vault as one of the S
-=== Configuration
-
-NOTE: If you choose to use a security principal to authenticate and authorize with Azure Active Directory for accessing an Azure resource, please refer to link:index.html#authorize-access-with-azure-active-directory[Authorize access with Azure AD] to make sure the security principal has been granted the sufficient permission to access the Azure resource.
-
-.Configurable properties of spring-cloud-azure-starter-keyvault-secrets
-[cols="2*", options="header"]
-|===
-|Property |Description
-| *spring.cloud.azure.keyvault.secret*.endpoint                                            | Key Vault uri
-| *spring.cloud.azure.keyvault.secret*.service-version                                     | Service version
-| *spring.cloud.azure.keyvault.secret*.property-source-enabled                             | Enable this property source
-| *spring.cloud.azure.keyvault.secret*.property-sources                                    | Multiple property source
-| *spring.cloud.azure.keyvault.secret*.property-sources[].name                             | Name of this property source
-| *spring.cloud.azure.keyvault.secret*.property-sources[].endpoint                         | Key Vault uri
-| *spring.cloud.azure.keyvault.secret*.property-sources[].service-version                  | Service version
-| *spring.cloud.azure.keyvault.secret*.property-sources[].case-sensitive                   | Whether the secret name is case-sensitive
-| *spring.cloud.azure.keyvault.secret*.property-sources[].secret-keys                      | The supported secret names. If not configured, it will retrieve all secret names.
-| *spring.cloud.azure.keyvault.secret*.property-sources[].refresh-interval                 | Refresh interval
-|===
+==== Basic Usage
 
-=== Basic Usage
-
-==== One Property Source
-
-===== Property Configuration
 If you want to authenticate by `client-id` and `client-secret`, the following properties are required:
 
+===== Configuration Properties
 [source,yml]
 ----
 spring:
   cloud:
     azure:
-      profile:
-        tenant-id: ${AZURE_TENANT_ID}
-      credential:
-        client-id: ${AZURE_CLIENT_ID}
-        client-secret: ${AZURE_CLIENT_SECRET}
       keyvault:
         secret:
-          property-source-enabled: true
-          endpoint: ${AZURE_KEYVAULT_ENDPOINT}
+          property-sources:
+            - name: key-vault-property-souece-1
+              endpoint: ${ENDPOINT_1}
+            - name: key-vault-property-souece-2
+              endpoint: ${ENDPOINT_2}
 ----
 
-If your application is authenticated by other methods like Managed Identity or Azure CLI, properties like `tenant-id`, `client-id`, `client-secret` are not necessary. But if these properties are configured, then these properties have higher priority. Please refer to link:authentication.html[Authentication section] to get more information.
-
-===== Java Code
-
+===== Java code
 [source,java]
 ----
+package com.azure.spring.keyvault.secrets.sample.single.property.source;
```

Comment:
Since we don't include package and imports in this whole reference doc, I think we should remove them here.

Reply:
Will do.